airbyte-integrations/connectors/source-unleash/source_unleash/spec.yaml

```
@@ -0,0 +1,33 @@
+documentationUrl: https://docs.airbyte.io/integrations/sources/unleash
+connectionSpecification:
+  $schema: http://json-schema.org/draft-07/schema#
+  title: Unleash Source Spec
+  type: object
+  required:
+    - api_token
+    - api_url
+  additionalProperties: true
+  properties:
+    api_token:
+      title: API Token
+      type: string
+      description: >-
+        Your API Token. See <a
+        href="https://docs.getunleash.io/how-to/how-to-create-api-tokens/">here</a>. The token is
+        case sensitive.
+      airbyte_secret: true
```

Please add examples to the `api_token` object:

```suggestion
      airbyte_secret: true
      examples:
        - "project:environment.be44368985f7fb3237c584ef86f3d6bdada42ddbd63a019d26955178"
        - "*:environment.be44368985f7fb3237c584ef86f3d6bdada42ddbd63a019d26955178"
        - "be44368985f7fb3237c584ef86f3d6bdada42ddbd63a019d26955178"
```

I sourced the sample tokens from their [quickstart](https://docs.getunleash.io/tutorials/quickstart#connecting-a-client-side-sdk-to-your-instance):
> Depending on whether you have the environments feature enabled or not, the API key will look a little different. If you don't have environments enabled, it'll just be a 64 character long hexadecimal string (for instance be44368985f7fb3237c584ef86f3d6bdada42ddbd63a019d26955178). If you do have environments enabled, the key will be prefixed with the project and the environment that the key is valid for. It'll use the format <project>:<environment>.<key>, e.g. demo-app:production.be44368985f7fb3237c584ef86f3d6bdada42ddbd63a019d26955178.